### ISSUE TITLE
[Music Replacement] Add a column showing the LI1 Cassetes-v3 filename of the track?

### ISSUE THREAD

**Issue by wertercatt** [itsmattkc/LEGOIslandRebuilder#31]
When attempting to replace the music like you did in your video (_How I Replaced the Music in LEGO Island_ (2019)) it's unclear what filenames from the higher quality transfers correspond to which in-game filenames.

**Comment by itsmattkc** [comment]
IMO it would be to have a pack with the original `JUKEBOX.SI` filenames (which is what Rebuilder is showing) rather than adhering to the arbitrary names used in the pack

**Comment by wertercatt** [comment]
I agree, but I'm not familar enough with the music to rename the files from projectisland.org to their JUKEBOX.SI names.

**Comment by angela-dayo** [comment]
In case anyone else comes across this thread, I ended up just manually matching up each individual song — so here's a _really_ low-effort copy-paste of the filenames that I figured out and matched up.

These filenames have been slightly modified by me (mostly just noting down discrepancies I noticed compared to the original audio), but you should be able to tell which ones they are from the [Project Island high-quality music page](https://projectisland.org/music/).

**※ NOTE 1:** The "Collection" \*.zip file that is available on the Project Island page is out of date — I recommend downloading each file individually.

**※ NOTE 2:** There's some trailing silence at the beginning of the version of "The Brickster - Decal" found on the Project Island page — you can remove that with an audio editor (hence the "[RETIMED]" tag I gave my version below).

**※ NOTE 3:** A few of these files are in FLAC and MP3 formats — you can convert them to WAV using a tool such as `ffmpeg`. (I personally just ran `for file in *.mp3 *.flac; do ffmpeg -i "${file}" "${file}.wav"` in a Bash shell to convert all of them at once.)

---

## Project Island filenames to JUKEBOX.SI filenames

**※ NOTE:** Some of these songs have multiple versions available on Project Island — check the "Files that are not recommended" section below to know which one to pick!

* `Baroque in Brick (JBMusic5, theme_3).wav`
* `Brick By Brick - Restored Version 2019-08 (JBMusic2, towtruck).wav`
* `Chase Theme (BrickstrChase, chase).wav`
* `Chasing the Brickster [HIGHER KEY] (BrickHunt, brikhunt).wav`
* `Dune Buggy Build (Brick By Brick) (DuneCarBuild_Music, Dunebld).mp3.wav`
* `Fanfare Theme (ResidentialArea_Music, Theme_2).wav`
* `Happy Roaming - Restored Version (CentralRoads_Music, lego_3).wav`
* `Helicopter Chase [LOWER KEY] (Act3Music, Lego_2).mp3.wav`
* `Information Center [LOWER KEY, SLOWER TEMPO] (InformationCenter_Music, lego_1).mp3.wav`
* `Jail Theme (Jail_Music, jail AND MusicTheme3, 8bjail1).wav`
* `LEGO Island Theme-01 (BeachBlvd_Music, Theme_1).wav`
* `Mama Papa Brickolini (JBMusic1, mamapapa AND MusicTheme1, Pasqua22).wav`
* `Observation Deck (InfoCenter_3rd_Floor_Music, lego_5).mp3.wav`
* `Park Theme (Park_Music, park).wav`
* `Police Station [HIGHER KEY] (PoliceStation_Music, pol_sta).wav`
* `The Brickster - Decal [RETIMED] (JBMusic3, briktost).wav`
* `The Race (RaceTrackRoad_Music, racing).wav`
* `The Torpedoes - Interplanetary (Beach_Music, 2_interp).flac.wav`
* `The Torpedoes - Manta_Ray [CORRECT KEY] (JBMusic6, 4_mantar).flac.wav`
* `The Torpedoes - Tarantula (JetskiRace_Music, 3_tarant).flac.wav`
* `The Torpedoes - Torpedoe [SLOWER TEMPO] (JetskiBuild_Music, jetbld).flac.wav`
* `Think I'm Gonna Build Me a Bridge (JBMusic4, country).wav`

---

## Files that are not recommended
* `Brick by Brick (JBMusic2, towtruck).wav` — use the restored version instead.
* `Cave Theme [MISSING PERCUSSION] (Act2Cave, cave).wav` — the percussion is missing entirely.
* `Happy Roaming (CentralRoads_Music, lego_3).mp3` — use the restored version instead.
* `Happy Roaming (CentralRoads_Music, lego_3).wav` — use the restored version instead.
* `Information Center [HIGHER KEY] (InformationCenter_Music, lego_1).wav` — the MP3 version is mixed better.
* `The Torpedoes - Manta Ray [HIGHER KEY] (JBMusic6, 4_mantar).wav` — use the FLAC version instead, which has the correct key and doesn't have a fade-in at the beginning.

---

## Files that don't seem to exist in-game
* `Brief Brickster Theme (UNKNOWN).wav`
* `In The Box Polka (UNKNOWN).wav`
* `Legomania-v4 (UNKNOWN).wav`
